inject/build.gradle

```diff
@@ -11,11 +22,12 @@ micronautBuild {
 }
 
 dependencies {
-    compileOnly libs.javax.inject
     api libs.jakarta.inject.api
     api libs.managed.jakarta.annotation.api
     api project(':core')
+    api libs.jetbrains.annotations
```

Comment:
this is a source level only dependency, it shouldn't be needed

Reply:
yeah my bad - fixed jetbrains annotations and added comments to explain it